app/gui/view/graph-editor/src/component/edge.rs

```diff
@@ -47,13 +47,15 @@ define_endpoints_2! {
         /// dragged by the mouse.)
         source_attached(bool),
         set_disabled(bool),
+        /// Whether the edge should stop responding to mouse movement.
+        set_hover_disabled(bool),
```

Comment:
Is this ever different from `!source_attached || !target_attached`?

Reply:
Yes. The hover is disabled for all edges whenever there is any detached edge present. It is disabled notonly for the edge that is actually detached.